pkg/oam/labels.go

```diff
@@ -211,6 +211,9 @@ const (
 	// AnnotationControllerRequirement indicates the controller version that can process the application.
 	AnnotationControllerRequirement = "app.oam.dev/controller-version-require"
 
+	// AnnotationDefinitionControllerRequirement indicates the controller version that can process the definition.
+	AnnotationDefinitionControllerRequirement = "definition.oam.dev/controller-version-require"
```

Comment:
let's use the same annotation `AnnotationControllerRequirement`, can we?

Reply:
`AnnotationControllerRequirement` is used for application CR

Reply:
can they be the same one?

Reply:
v.1.2.x already uses this feature, if it is changed to the same name, there will be incompatibility problems